### PR TITLE
Fix c bug

### PR DESCRIPTION
**Pull request by sam-halfpenny**
fixed

### CODE REVIEW

File: claustrophobia/claustrophobia.js [hunk 1]
```diff
@@ -350,7 +364,7 @@ function gameloop(timestamp) {
     grid()
     sheepdog1.draw(ctx)
     sheepdog1.update()
-    if(!kill && !end){
+    if(!end && !dead){
```
Comment (will-0) [inline claustrophobia/claustrophobia.js:367]: There's no reference to dead in your file. Additionally, switching it back to what it used to be randomly makes you win.

**Review by will-0 — APPROVED** [review]
Randomly dying

File: claustrophobia/claustrophobia.js (hunk 1 above)
Comment (will-0) [inline claustrophobia/claustrophobia.js:367]: There's no reference to dead in your file. Additionally, switching it back to what it used to be randomly makes you win.

**Review by will-0 — CHANGES_REQUESTED** [review]

**Comment by sam-halfpenny** [comment]
i have added hole dodger on to this branch to avoid git conflicts porque no me gusta :sunglasses:

File: claustrophobia/claustrophobia.js (hunk 1 above)
Comment (sam-halfpenny) [reply claustrophobia/claustrophobia.js]: you will win randomly if all the sheep randomly wander into the red square

**Comment by sam-halfpenny** [comment]
sorry i figured out what was wrong, i had implanted a kill switch to prevent my computer from exploding when i wanted to log data every frame and i forgot to delete it

**Review by will-0 — APPROVED** [review]
I literally cannot beat this game even when I set it to 57 (which is one square missing)

But it seems to work. Nice job man x

**Pull request merged**